101-anf-nfs-volume/azuredeploy.json

```diff
@@ -0,0 +1,186 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
add metadata descriptions for all parameters

Reply:
added